src/Appwrite/Platform/Modules/Locale/Http/Locale/Get.php

```
@@ -0,0 +1,89 @@
+        $currency = null;
+
+        $record = $geodb->get($ip);
+
+        if ($record) {
+            $output['countryCode'] = $record['country']['iso_code'];
+            $output['country'] = $locale->getText('countries.' . \strtolower($record['country']['iso_code']), $locale->getText('locale.country.unknown'));
+            $output['continent'] = $locale->getText('continents.' . \strtolower($record['continent']['code']), $locale->getText('locale.country.unknown'));
+            $output['continentCode'] = $record['continent']['code'];
+            $output['eu'] = (\in_array($record['country']['iso_code'], $eu)) ? true : false;
+
+            foreach ($currencies as $code => $element) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$code'. (undefined)

(UnusedLocalVariable)